examples/simple.rs

```diff
@@ -0,0 +1,74 @@
+use astarte_sdk::AstarteOptions;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Cli {
+    // Realm name
+    #[structopt(short, long)]
+    realm: String,
+    // Device id
+    #[structopt(short, long)]
+    device_id: String,
+    // Credentials secret
+    #[structopt(short, long)]
+    credentials_secret: String,
+    // Pairing URL
+    #[structopt(short, long)]
+    pairing_url: String,
+    // Interfaces directory
+    #[structopt(short, long)]
+    interfaces_directory: String,
+}
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+
+    let Cli {
+        realm,
+        device_id,
+        credentials_secret,
+        pairing_url,
+        interfaces_directory,
+    } = Cli::from_args();
+
+    let mut sdk_options =
+        AstarteOptions::new(&realm, &device_id, &credentials_secret, &pairing_url);
+
+    sdk_options
+        .add_interface_files(&interfaces_directory)
+        .unwrap();
+
+    sdk_options.build().await.unwrap();
+
+    let mut device = sdk_options.connect().await.unwrap();
+
+    let w = device.clone();
+    tokio::task::spawn(async move {
+        let mut i: f64 = 0.0;
+        loop {
+            w.send("com.test.everything", "/double", i).await.unwrap();
```

Comment:
let's use same naming conventions everywhere: `"com.test.everything"` -> "com.test.Everything"

Reply:
Also we might use https://raw.githubusercontent.com/astarte-platform/astarte/master/standard-interfaces/org.astarte-platform.genericsensors.Values.json that is always good for our examples

Reply:
I'd like to keep com.test.everything because it tests all the data types, I can remove the other one